docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -0,0 +1,351 @@
+{
```

Comment:
<p>Since this is a new tutorial, I don't think we need to mention both interfaces - it's unnecessarily distracts the reader from the important thing, the progress bar (+ the progress_fn is passed the same way).</p><p>I'd prefer to always use the class interface the functional interface is just a wrapper and I would mention in the text that that:</p><p>... the same applies for {func}<code>~ott.solvers.linear.sinkhorn.solve</code>,as shown in the {doc}<code>basic_ot_between_datasets</code> tutorial. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I respectfully disagree ;) I think covering both interfaces may really help newcomers.</p><p> </p><p>I rewrote the examples section, for clarity. Let me know what you think with this new version, and then ok, I'll remove it if you prefer.</p>

Reply:
Ok, let's keep it as it is then, don't have really strong opinion.
Could you just change 1 thing: I'd remove the numbering, e.g., `1. Tracking progress` -> `Tracking progress` and `a. With the basic callback function` -> `With the basic callback function`, don't think it's really needed (the different heading levels should be enough).